.luarc.json

```diff
@@ -1,24 +1,53 @@
 {
   "$schema": "https://raw.githubusercontent.com/sumneko/vscode-lua/master/setting/schema.json",
-  "runtime.version": "Lua 5.1",
   "completion.requireSeparator": "/",
+  "diagnostics": {
+    "globals": [
+      "VFS",
+      "Spring",
+      "Game",
+      "CMD",
+      "gadget",
+      "widget",
+      "GG",
+      "WG",
+      "gadgetHandler",
+      "widgetHandler"
+    ],
```

Comment:
These should not be added, there is no need for them anymore after the linter and Recoil library submodule were put in place

Reply:
Done